src/app/Archives/AllTargetsArchivedRecordingsTable.tsx

```diff
@@ -315,21 +332,23 @@ export const AllTargetsArchivedRecordingsTable: React.FC<AllTargetsArchivedRecor
     setSearchText('');
   }, [setSearchText]);
 
+  const targetDisplay = React.useCallback(
+    (target: Target) =>
+      target.alias == target.connectUrl || !target.alias
+        ? `${target.connectUrl}`
+        : `${target.alias} (${target.connectUrl})`,
```

Comment:
This second format (the one including the alias) could be extracted out to a localization as well. I suppose that for right-to-left languages it would make sense for the order of these two to be switched as well.